stdlib/public/core/Integers.swift.gyb

```diff
@@ -1898,6 +1906,26 @@ extension BinaryInteger {
 }
 
 //===----------------------------------------------------------------------===//
+// FIXME(integers): remove these extensions once constrained Words is fast
+// enough
+extension BinaryInteger {
+  @_transparent
+  public var _countRepresentedWords: Int {
+    return (self.bitWidth + ${word_bits} - 1) / ${word_bits}
+  }
+}
+
+extension BinaryInteger where
+  Words : Collection,
+  Words.Index : BinaryInteger,
+  Words.Element == UInt {
+  public func _word(at n: Int) -> UInt {
+    return words[numericCast(n)]
```

Comment:
There should be a range check here; this won't correctly handle the `n >= words.endIndex` case.
Also, `words.startIndex` might be nonzero in some unusual but (probably) legitimate implementations.

Reply:
Thanks.